tlock_test.go

```diff
@@ -92,6 +93,53 @@ func TestEarlyDecryptionWithRound(t *testing.T) {
 	require.ErrorIs(t, err, tlock.ErrTooEarly)
 }
 
+func TestEncryptionWithTimestamp(t *testing.T) {
+	if testing.Short() {
+		t.Skip("skipping live testing in short mode")
+	}
+
+	network, err := http.NewNetwork(testnetHost, testnetChainHash)
+	require.NoError(t, err)
+
+	// =========================================================================
+	// Encrypt
+
+	// Read the plaintext data to be encrypted.
+	in, err := os.Open("test_artifacts/data.txt")
+	require.NoError(t, err)
+	defer in.Close()
+
+	// Write the encoded information to this buffer.
+	var cipherData bytes.Buffer
+
+	// Timestamp to duration
+	rand.Seed(time.Now().UnixNano())
```

Comment:
I'm not sure seeding `rand` is necessary, as `age` will pull it from a local entropy source anyway